### DISCUSSION TITLE
After upgrading to 0.41.1 zellij throws an error

### DISCUSSION THREAD

**Discussion by mrwsl** [zellij-org/zellij#3739] [Q&A] [answered]
I am pretty sure, that it is me not understanding the breaking changes. After upgrading from 0.40.0 to 0.41.1 I get the following error:
```
ERROR  |zellij_server::plugins::w| 2024-11-07 07:01:42.216 [async-std/runti] [zellij-server/src/plugins/wasm_bridge.rs:712]: Failed to convert to protobuf: "unsupported key modifier" 
```
I noticed that the `--layout` command works different now, but still can't make it work with `--new-session-with-layout`.
Here is a screencast of what is happening:

https://github.com/user-attachments/assets/5ba2426b-a5a5-45f2-a16f-828704141b8e

The "Resurrect Session" tab is empty which it is not in version 0.40.0 and I can't use CTRL+Q to exit zellij. Any hints on what I have missed to change after upgrading? Thanks!

**Comment by imsnif** [comment, marked as answer]
Hey, the video is not showing on this page so I'm not 100% certain what's going on, but there's a fix in main for the "unsupported modifier" error that might help (again - not sure if that's what we're seeing here though).

**Reply by mrwsl** [reply]
I will give it a try. Disabling Numlock did the trick for now!
